README.md

```diff
@@ -5,14 +5,19 @@ Explore large, annoying graphs using hierarchies of dominating sets - because
 in space, no one can hear you miao!
 
 This is a collaboration between the
-[Theory In Practice](https://github.com/TheoryInPractice/) lab at NC
-State and the
-[Lab for Data Intensive Biology](https://github.com/dib-lab/) at UC
-Davis, generously supported by the Moore Foundation's
+[Theory In Practice](https://github.com/TheoryInPractice/) lab at University of Utah, the
+[Lab for Data Intensive Biology](https://github.com/dib-lab/) at UC Davis, and
+[Dr. Felix Reidl](https://www.dcs.bbk.ac.uk/about/people/academic-staff/felix/) at Birkbeck University of London,
+ generously supported by the Moore Foundation's
```

Comment:
Initial development of spacegraphcats was supported by...

(this also reflects the expansion to Felix's lab, which is not Moore funded)